pygac/tests/test_reader.py

```diff
@@ -192,9 +192,50 @@ def test_get_tle_lines(self, read_tle_file, *mocks):
                 self.assertEqual(tle1, tle_data[tle_idx])
                 self.assertEqual(tle2, tle_data[tle_idx + 1])
 
+    @mock.patch('pygac.gac_reader.GACReader._get_corrupt_mask')
+    def test_get_angles(self, get_corrupt_mask):
+        """Test get_angles function of the reader."""
+        # Line: 1, 649, 6198 and 12658 from Tiros-N file (1980-01-03 11:47)
+        lon_i = np.array(
+            [69.41555, 152.10587, 164.3131, 67.23855])[:, np.newaxis]
+        lat_i = np.array(
+            [71.6283, 85.24265, -62.076958, 82.72296])[:, np.newaxis]
+        get_corrupt_mask.return_value = np.isnan(lon_i)
+        self.reader.lons = lon_i
+        self.reader.lats = lat_i
+        self.reader.tle_lines = [
+            '1 11060U 78096A   80003.54792075  .00000937  00000-0  52481-3 0  2588\r\n',
+            '2 11060  98.9783 332.1605 0012789  88.8047 271.4583 14.11682873 63073\r\n']
+        self.reader.utcs = np.array(
+            [315748035469, 315748359969,
+             315751135469, 315754371969]).astype('datetime64[ms]')
+        self.reader.spacecrafts_orbital = {25: 'tiros n'}
+        self.reader.spacecrafts_names = {25: 'tirosn'}
```

Comment:
There is no reader with a `spacecrafts_names` attribute, so I guess this can be removed.

Reply:
Ok.